infrastructure/logging/src/main/java/tech/pegasys/teku/infrastructure/logging/EventLogger.java

```diff
@@ -152,22 +152,18 @@ public void executionClientRequestTimedOut() {
         Color.YELLOW);
   }
 
-  public void executionClientRecovered() {
-    info("Execution Client request succeeded after a previous failure", Color.GREEN);
-  }
-
-  public void builderIsOffline(final String errorMessage) {
-    String builderOfflineEventLog =
+  public void builderIsNotAvailable(final String errorMessage) {
+    final String builderIsNotAvailableEventLog =
         String.format(
-            "The builder is offline: %s. Block production will fallback to the execution engine.",
+            "The builder is not available: %s. Block production will fallback to the execution engine.",
             errorMessage);
-    warn(builderOfflineEventLog, Color.YELLOW);
+    warn(builderIsNotAvailableEventLog, Color.YELLOW);
   }
 
-  public void builderIsBackOnline() {
-    String builderOnlineEventLog =
-        "The builder is back online. It will be used for block production.";
-    info(builderOnlineEventLog, Color.GREEN);
+  public void builderIsAvailable() {
+    final String builderIsAvailableEventLog =
+        "The builder is available. It will be used for block production.";
```

Comment:
should probably call out these messages have been changed - if anyone is monitoring for them they'll be missing in their old format...